context-propagation/pom.xml

```diff
@@ -54,6 +54,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-postgresql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-web</artifactId>
```

Comment:
What's the reason ?

Reply:
WebClient and `response.bodyAsJsonObject`